navigator/frontend/src/ui-core/src/ArgumentDisplay/index.tsx

```diff
@@ -99,6 +99,18 @@ const ArgumentDisplay = (props: Props): JSX.Element => {
     }
     case 'bool': return <span>{argument.value ? 'TRUE' : 'FALSE'}</span>;
     case 'unit': return <span>unit</span>;
+    case 'map' : return (
+      <NestedForm level={level}>
+        {argument.value.length > 0 ? argument.value.map((entry, _) => (
+          <LabeledElement key={entry.key} label={`${entry.key}`} className={className}>
```

Comment:
```suggestion
          <LabeledElement key={entry.key} label={entry.key} className={className}>
```
No need for string interpolation here.